src/getAppSyncConfig.js

```diff
@@ -199,13 +209,15 @@ export default function getAppSyncConfig(context, appSyncConfig) {
 
       const requestTemplate = request || `${defaultTemplatePrefix}.request.vtl`;
       const responseTemplate = response || `${defaultTemplatePrefix}.response.vtl`;
+      const defaultRequestTemplate = cfg.defaultMappingTemplates.request;
+      const defaultResponseTemplate = cfg.defaultMappingTemplates.response;
```

Comment:
We need to be careful here.

In sid88in/serverless-appsync-plugin the order of preference is:
- request/response from the template
- default request/response template
- [Parent].[field].request/response/vlt

Here, the strategy is different. Order is
- request/response from the template
- [Parent].[field].request/response.vlt

With a fallback on default if the file does not exist.

To avoid surprises at deployment time, I'd prefer to keep the same logic as in the plugin.

```suggestion
      const requestTemplate = request || cfg.defaultMappingTemplates.request || `${defaultTemplatePrefix}.request.vtl`;
      const responseTemplate = response || cfg.defaultMappingTemplates.response || `${defaultTemplatePrefix}.response.vtl`;
```

and leave the rest unchanged. 

Am I missing something?

Reply:
hahahah. So much better! Thank you for reviewing!